schemas/EconomicEvent.json

```diff
@@ -1,71 +1,107 @@
+            "description": "Specific time marking the exact time at which the economic event occurred.",
             "type": "string",
             "format": "date-time"
         },
         "before": {
-            "description": "The economic event occurs before this date/time.",
+            "description": "Indicates the  economic event occurred prior to this specific time.",
             "type": "string",
             "format": "date-time"
         },
         "after": {
-            "description": "The economic event occurs after this date/time.",
+            "description": "Indicates the economic event occurred after this specific time.",
             "type": "string",
             "format": "date-time"
         },
+        "atLocation": {
+            "description": "The place where an economic event occurs.  Usually mappable.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/SpatialThing.json"
+        },
         "note": {
             "description": "A textual comment or description.",
             "type": "string"
         },
         "inputOf": {
-            "description": "The process for which this economic event describes an input.",
-            "$ref": "https://raw.githubusercontent.com/valueflows/vf-json-schema/master/schemas/Process.json"
+            "description": "Defines the process to which this event is an input.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/Process.json"
         },
         "outputOf": {
-            "description": "The process for which this economic event describes an output.",
-            "$ref": "https://raw.githubusercontent.com/valueflows/vf-json-schema/master/schemas/Process.json"
+            "description": "Defines the process for which this event is an output.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/Process.json"
+        },
+        "triggeredBy": {
+            "description": "References another economic event that implied this economic event, often based on a prior agreement.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/EconomicEvent.json"
+        },
+        "realizationOf": {
+            "description": "This economic event occurs as part of this agreement.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/Agreement.json"
         },
-        "under": {
-            "description": "Reference to an agreement between agents which specifies the rules or policies which govern this event.",
-            "$ref": "https://raw.githubusercontent.com/valueflows/vf-json-schema/master/schemas/Agreement.json"
+        "agreedIn": {
+            "description": "Reference to an agreement between agents which specifies the rules or policies or calculations which govern this economic event.",
```

Comment:
The word "calculation" is probly used in other agreement references, but this is the first time it triggered the question of what does that mean?

Reply:
refers to for example values equations / distribution agreements - the calculation explains why you see that in a claim